### PR TITLE
NONAME-152 adding and removing users from existing groups

### PR DESCRIPTION
**Pull request by K4RP4T**

### CODE REVIEW

File: frontend/src/lib/components/MultiSelect.svelte [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 <MultiSelect
 	bind:selected
-	{options}
+	options={options.toSorted()}
```
Comment (yanvoi) [inline frontend/src/lib/components/MultiSelect.svelte:12]: what does this achieve?

Comment (K4RP4T) [reply frontend/src/lib/components/MultiSelect.svelte:12]: when e-mails are sorted alphabetically, then users can easily find the emails they are looking for

Comment (yanvoi) [reply frontend/src/lib/components/MultiSelect.svelte:12]: good touch, approved

**Review by yanvoi — APPROVED** [review]
but please do wait for someone else to also approve the changes

**Review by sjerzykiewicz — APPROVED** [review]

**Pull request merged**